app/views/content/train-to-be-a-teacher/initial-teacher-training.md

```diff
@@ -79,6 +78,9 @@ School placements vary in length and structure, but the minimum length is usuall
 
 Read about [what teacher training is like doing a QTS with PGCE with one training provider](/blog/what-to-expect-on-your-teacher-training).
 
+## Holidays
```

Comment:
I think you are right - this is confusing as we don't have the info for non-PGCE courses. Let's remove for now and look at enhancing the page properly under a future ticket